src/nvim/memory.c

```diff
@@ -42,8 +42,6 @@ void try_to_free_memory(void)
   clear_sb_text();
   // Try to save all buffers and release as many blocks as possible
   mf_release_all();
-  // cleanup recursive lists/dicts
-  garbage_collect();
```

Comment:
I think that this part should be present in the same commit which marks Vim patch 916 present.